src/App.tsx

```diff
@@ -1,11 +1,26 @@
 import './App.css'
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import LandingPage from './pages/LandingPage'
+import withAuth from './hocs/WithAuth/WithAuth'
+import Dashboard from './pages/Dashboard';
+
 
 function App() {
-  
+  //dashboard isnt real yet
+  const AuthenticatedDashboard = withAuth(Dashboard);
 
   return (
-    <LandingPage />
+    <Router>
+      <Routes>
+        {/*public routes*/}
+        <Route path="/" element={<LandingPage />} />
+        {/* protected routes*/}
+        <Route
+          path="/dashboard"
+          element={<AuthenticatedDashboard />}
```

Comment:
might want to do something like or go wrap the export of dashboard in with auth at the bottom of file of there will be a bunch of these variables
```suggestion
  return (
    <Router>
      <Routes>
        {/*public routes*/}
        <Route path="/" element={<LandingPage />} />
        {/* protected routes*/}
        <Route
          path="/dashboard"
          element={<AuthenticationGuard component={Dashboard} />}
          ```